hasura/graphql-engine
# avoid setting up hdb_views schema on schema sync refreshing cache

### Description



### Affected components 


- Server

### Related Issues




### Solution and Design


- While running multiple HGE instances on the same database, If a server instance receives metadata update event from other instance then rebuild its schema cache without clearing and setting up `hdb_views` schema. 
- Clear `hdb_views` schema's views and routines in order in SQL `for loop`.

### Steps to test and verify


- Run more than 2 HGE instances on the same database
- Create insert permission on one of the server instance
- Check logs on other two for successful schema cache reload
